src/components/CategoricalDisplay.jsx

```diff
@@ -1,6 +1,7 @@
 import React, { Component, Fragment } from 'react';
 import {CategoryIconMap, ItemIconMap, SearchIcon,
     SymbolPlaceholderIcon, ActiveOptionsIconMap, FavoriteIcon, CloseIcon } from './Icons.jsx';
+import {stxtap} from '../store/utils';
```

Comment:
Changes in this PR to `CategoricalDisplay.jsx` are outdated; you'll need to revert this. It's always best to handle merge conflicts manually or with a tool (e.g. 3-way merging)

Reply:
Sure. will do.